script/generate-documentation

```diff
@@ -57,7 +57,7 @@ update_docs() {
             [[ -z "${CIRCLE_SHA1}" ]] || (cd .. && git log --pretty=fuller "${CIRCLE_SHA1}" -1 >> out/last.commit)
             git commit -F last.commit
             if [[ -n "${PUBLISH}" ]] && [[ "${PUBLISH}" != 0 ]]; then
-                git push "$TARGET_REMOTE" "$TARGET_BRANCH"
+                git push -q -f "$TARGET_REMOTE" "$TARGET_BRANCH"
```

Comment:
What is `-q -f` meant to accomplish here? :thinking:

Reply:
Might be worth adding a comment to explain that actually, and not just a reply here